airflow/www/static/js/tree/api/useTasks.js

```diff
@@ -27,5 +27,8 @@ export default function useTasks(dagId) {
   return useQuery(
     ['tasks', dagId],
     () => axios.get(tasksUrl),
+    {
+      placeholderData: { tasks: [] },
```

Comment:
Is this argument a default return value for the `useQuery` function?

Reply:
Yes, but it doesn't affect the cache https://react-query.tanstack.com/guides/placeholder-query-data